sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -130,6 +131,8 @@ internal class RadarApiClient(
         if (usage != null) {
             queryParams.append("&usage=${usage}")
         }
+        val clientSdkConfiguration = RadarSettings.getClientSdkConfiguration(context).toString()
+        queryParams.append("&clientSdkConfiguration=${URLEncoder.encode(clientSdkConfiguration, "utf-8")}")
```

Comment:
just to confirm this is consistent with the iOS encoding?
Also a GET request can **technically** have a body, would we want to utilize that here?